mlir/include/mlir/Dialect/MIOpen/AffineMapHelper.h

```diff
@@ -58,9 +58,19 @@ inline bool hasPadding(AffineExpr expr) {
   };
   auto binaryExpr = expr.template dyn_cast<AffineBinaryOpExpr>();
   if (binaryExpr) {
-    ret |= hasMinusConstant(binaryExpr.getLHS());
-    if (ret)
+    AffineExpr tmp = binaryExpr.getLHS();
+
```

Comment:
Do we also need to check the RHS?

if yes, the codes need to be revised
If no, perhaps a comment is needed to document why the RHS doesn’t need such a check